test/salt/pillar/postgres.sls

```diff
@@ -18,7 +18,7 @@ postgres:
   {%- if not (grains.os_family == 'Debian') %}
   version: '9.6'
   {%- else %}
-  version: '10'
+  version: '13'
   {%- endif %}
```

Comment:
[Just checked](https://gitlab.com/myii/postgres-formula/-/pipelines/315779405), we no longer need the `if` block:

```suggestion
  version: '13'
```